api/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/cespare/xxhash v1.1.0
 	github.com/coocood/freecache v1.1.1
 	github.com/fatih/color v1.9.0
-	github.com/feast-dev/feast/sdk/go v0.9.2
+	github.com/feast-dev/feast/sdk/go v0.9.4
```

Comment:
Shouldn't we be using >0.9.7 where pyyaml vulnerability is fixed?

Reply:
That one is for python sdk. But for golang sdk, it's 0.9.4